src/Service/EnterpriseSearchService.php

```diff
@@ -534,14 +571,14 @@ private function handleError(?array $responseBody): void
      */
     private function getSchemaForFields(array $fields): SchemaUpdateRequest
     {
-        $definedSpecs = new SchemaUpdateRequest();
+        $request = Injector::inst()->create(SchemaUpdateRequest::class);
```

Comment:
this looks like it is using `new` in the tests so should probably be updated there too